scripts/ci/diff.sh

```diff
@@ -348,31 +428,31 @@ elif [ "$(echo "$post_condition" | jq '.has_diff')" = "true" ] && [ -n "$diff_re
   echo "Posting comment as post_condition is set to has_diff and there is a diff"
 elif [ -z "$percent" ]; then
   echo "Posting comment as percentage diff is empty"
-elif [ $(echo "$absolute_percent > $percentage_threshold" | bc -l) = 1 ]; then
+elif [ "$(echo "$absolute_percent > $percentage_threshold" | bc -l)" = 1 ]; then
   echo "Posting comment as percentage diff ($absolute_percent%) is greater than the percentage threshold ($percentage_threshold%)."
 else
   echo "Not posting comment as percentage diff ($absolute_percent%) is less than or equal to percentage threshold ($percentage_threshold%)."
   cleanup
   exit 0
 fi
 
-if [ ! -z "$GITHUB_ACTIONS" ]; then
+if [ -n "$GITHUB_ACTIONS" ]; then
   echo "::set-output name=past_total_monthly_cost::$past_total_monthly_cost"
   echo "::set-output name=total_monthly_cost::$total_monthly_cost"
   load_github_env
```

Comment:
Is there a reason why we call this twice? it's already called on line 362 (maybe @aliscott knows as he added that functionality). Same issue with `load_gitlab_env` below

Reply:
Don't think there's a reason for this.

Reply:
@tim775 it seems like we can remove the lines `load_github_env` and `load_gitlab_env` below